pkg/chunk/cached_store.go

```diff
@@ -42,6 +42,8 @@ const SlowRequest = time.Second * time.Duration(10)
 
 var (
 	logger = utils.GetLogger("juicefs")
+	UseMountUploadLimitConf = false
```

Comment:
It's better to add a `Reload()` interface for `chunk.Config` and hide these details inside chunk/

Because we may want to reload other fields of chunk.Config, and we don't add these for them.